resources/language/resource.language.es_es/strings.po

```diff
@@ -264,7 +264,7 @@ msgstr "diariamente"
 
 msgctxt "#30068"
 msgid "every other day"
-msgstr "cada dos días"
+msgstr "cualquier otro día"
```

Comment:
Sorry, but this translation is not correct. 'Every other day' is a **fixed expression** and my previous translation is the right one. Please see [https://www.wordreference.com/es/translation.asp?tranword=every%20other%20day](url)

Reply:
Ok

Reply:
Do we translate it as?? > 'en días alternos'